src/VKBindings.h

```diff
@@ -19,6 +19,7 @@ struct VKBind
     std::string DisplayName{};
     std::variant<std::string, std::function<void()>> Description{};
     std::variant<std::function<TVKBindHotkeyCallback>, std::function<TVKBindInputCallback>> Handler{};
```

Comment:
the variant should copy the enum, having Input, Hotkey and OverlayHotkey options, instead of boolean differentiating between hotkey and overlayhotkey

i know the order is a bit different now but swap it when you will change it

Reply:
also misses IsOverlayHotkey function and corresponding Lua callback (believe there was callback for it, will need to check)

i would also add probable IsValid, which would be true if variant is not std::monostate when you update it